manifests/kustomize/env/platform-agnostic-multi-user-legacy/kustomization.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../../base/installs/multi-user
+  - ../../base/metadata/overlays/db
```

Comment:
shall we add a comment here or a readme in this folder to clarify its purpose?
anyway, I think this is not a blocker you can submit a separate PR too.

Reply:
I was going to edit your branch directly to add this comment, but you didn't set the config to allow maintainers edit your branch. Would you be interested to do that next time? So I can help you edit some minor stuff

Reply:
> I was going to edit your branch directly to add this comment, but you didn't set the config to allow maintainers edit your branch. Would you be interested to do that next time? So I can help you edit some minor stuff

Thanks for the heads-up on this. I tried to find this button when creating a PR and noticed the following:
- If I make a PR from my own fork (`yanniszark/pipelines`), I can allow edits from maintainers.
- If I make a PR from my company's fork (`arrikto/kubeflow-pipelines`), then no such option is displayed.

Thus, I need to bring it up internally and see what we can do about it, seems it looks like a permissions issue. I agree that it would be helpful to enable.

Reply:
> shall we add a comment here or a readme in this folder to clarify its purpose?

I will create a separate PR for that as well.